src/api/app/models/token/workflow.rb

```diff
@@ -12,7 +12,7 @@ def call(options)
     scm_extractor_payload = extractor.call
     yaml_downloader = Workflows::YAMLDownloader.new(scm_extractor_payload)
     yaml_file = yaml_downloader.call
-    raise Token::Errors::NonExistentWorkflowsFile, yaml_downloader.errors.join('\n') if yaml_file.blank?
+    raise Token::Errors::NonExistentWorkflowsFile, "issues with workflows.yml: #{yaml_downloader.errors.join('\n')}" if yaml_file.blank?
```

Comment:
Maybe something like this? The specs should also be updated.

```suggestion
    raise Token::Errors::NonExistentWorkflowsFile, '".obs/workflows.yml" could not be downloaded on the branch "TARGET_BRANCH": #{yaml_downloader.errors.join('\n')}" if yaml_file.blank?
```